src/objects/Volunteers_Settings__c.object

```diff
@@ -49,6 +49,15 @@
         <type>Text</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>Grant_Guest_Users_Update_Access__c</fullName>
+        <defaultValue>false</defaultValue>
+        <externalId>false</externalId>
+        <inlineHelpText>Allow guest users to update records based on their Create access.</inlineHelpText>
```

Comment:
@katherinemitchell - This is the branch with the new custom setting. Please let me know if you would like additional changes. Thank you!